eureka-client/src/main/java/com/netflix/appinfo/providers/Archaius1VipAddressResolver.java

```diff
@@ -0,0 +1,39 @@
+package com.netflix.appinfo.providers;
+
+import com.netflix.config.DynamicPropertyFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Archaius1VipAddressResolver implements VipAddressResolver {
```

Comment:
This code repeats for Archaius1 and Archaius2.  Any thoughts on creating an AbstractVipAddressResolver to hold the common code or put this in a utility class.